src/components/RegisterForm/RegisterForm.jsx

```diff
@@ -1,69 +1,139 @@
-					placeholder="Введите компанию"
-					defaultValue=""
-					name="company"
-				/>
-				<div className={registerFormStyles.checkboxContainer}>
-					<input
-						className={registerFormStyles.checkbox}
-						id="register-checkbox"
-						type="checkbox"
-						name="checkbox"
-					/>
-					<label
-						className={registerFormStyles.inputLabel}
-						htmlFor="register-checkbox"
-					>
-						Запомнить меня
-					</label>
-				</div>
-				<div className={registerFormStyles.buttonsContainer}>
-					<a className={registerFormStyles.back} to="/login">
-						Назад
-					</a>
-					<button className={registerFormStyles.submit} type="submit">
-						Зарегистрироваться
-					</button>
-				</div>
-			</form>
-		</div>
-	);
+
+  const [isOpenReg, setOpenReg] = useState(true);
+
+  const isDisabled = false;
+
+  function onClickReg() {
+    setOpenReg(false)
+  }
+
+  function onClickBack() {
+    setOpenReg(true)
+  }
+
+  return (
+    <div>
+      {isOpenReg ?
+        <div
+          className={registerFormStyles.formContainerOpened}
+        >
+          <h1 className={registerFormStyles.formTitle}>Регистрация</h1>
+          <p className={registerFormStyles.formText}>Введите E-mail и пароль, чтобы авторизоваться</p>
+          <form className={registerFormStyles.form}>
+            <Input
+              name='company'
+              placeholder='Компания'
+            />
+            <Input
+              name='email'
+              placeholder='E-mail'
+            />
+            <Input
+              name='password'
+              placeholder='Пароль'
+            />
+            <Input
+              name='repeat password'
+              placeholder='Повторите пароль'
+            />
+            <section className={registerFormStyles.container}>
+              <div className={registerFormStyles.checkboxContainer}>
+                <input
+                  className={registerFormStyles.checkbox}
+                  id="register-checkbox"
+                  type="checkbox"
+                  name="checkbox"
+                />
+                <label
+                  className={registerFormStyles.inputLabel}
+                  htmlFor="register-checkbox"
+                >
+                  Запомнить меня
+                </label>
+              </div>
+              <button
+                className={registerFormStyles.forgetButton}
+                type='button'
+              >
+                Забыли пароль?
+              </button>
+            </section>
+            <div className={registerFormStyles.buttonsContainer}>
+              <button
+                className={registerFormStyles.submit}
+                type="submit"
+                onSubmit={onClickReg}
+                // type='button'
+                // onClick={onClickReg}
+                disabled={isDisabled}
+              >
+                Зарегистрироваться
+              </button>
+              <div className={registerFormStyles.spanContainer}>
```

Comment:
```suggestion
              <div className={registerFormStyles.orContainer}>
```
эти элементы только для слова "или", пусть это будет отражено в названии, а то спан слишком абстрактно